content/docs/legacy-context.md

```diff
@@ -0,0 +1,215 @@
+---
+id: legacy-context
+title: Legacy Context
+permalink: docs/legacy-context.html
+---
+
+> This API is deprecated as of React 16.3.
+> 
+> The API will be supported in all 16.x releases, but applications using it should migrate to the [new API](/docs/context.html). The experimental API lacked a safe mechanism to update context. Version 16.3 introduced a new context API that is more efficient and supports both static type checking and deep updates.
+
+## How To Use Context
+
+> This section documents a deprecated API. See the [new API](/docs/context.html).
```

Comment:
I think these notes are pretty redundant with the one at the head of the page. Probably can remove them?

Reply:
Probably so. My concern is that it's a long page and someone might get a direct link to a section without noticing the deprecation warning.

Reply:
It's possible. Definitely seems redundant for the first section though, since the two notes are only like 50 pixels apart. I don't feel as strongly about the following sections.